core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.component.html

```diff
@@ -114,6 +114,12 @@
                     <span>Merge Skill</span>
                   </div>
                 </div>
+                <div class="unassign-skill-box e2e-test-mobile-unassign-skill-button" *ngIf="skill.topicNames.length" (click)="unassignSkill(skill.id)">
```

Comment:
Why is this UI change necessary? Can you show us a screenshot of this change?

Reply:
The "unassing skill" option is missing in mobile options so just added it. 
Here is the ss:
<img width="451" alt="Screenshot 2024-06-27 at 12 10 13 PM" src="https://github.com/oppia/oppia/assets/124369508/21b5b409-b289-4812-b4e8-f54098f5b61e">

Reply:
Thank you for the screenshot and explanation!

Reply:
@Akhilesh-max The "Delete" option seems to be missing here in the mobile view.

Reply:
Topic manager does not have access to delete the skill, so it won't show up.

Reply:
Ah okay, thanks for checking! Please fix the empty space.